lib/bin/watch.coffee

```diff
@@ -35,7 +35,7 @@ if fs.existsSync(dockerignorePath)
 ignoredFilesList = _.map(ignoredFilesList, (line) ->
   line.replace('.', '\\.')
 )
-ignoredFilesRegex = '/' + ignoredFilesList.join('|') + '/'
+ignoredFilesRegex = new RegExp(ignoredFilesList.join('|'))
```

Comment:
@tonybaroneee I've been doing some testing to see if that's the case. It doesn't seem like it makes a difference.

```js
> a = new RegExp('a|b')
/a|b/
> 'aaa'.replace(a, '_')
'_aa'
> 'bbb'.replace(a, '_')
'_bb'

> b = new RegExp('a\|b')
/a|b/
> 'abcde'.replace(b, '_')
'_bcde'
> 'aaa'.replace(b, '_')
'_aa'
> 'bbb'.replace(b, '_')
'_bb'
```